src/library/templates/library/library_bookshare_import.html

```diff
@@ -0,0 +1,85 @@
+                <form method="post">
+                    <div class="form-group">
+                        {% csrf_token %}
+                        <p>Bookshare download:</p>
+                        <ul>
+                            <li>Title: {{ title.title }}</li>
+                            {% for author in title.clusive.authors %}
+                            <li>Author: {{ author }}</li>
+                            {% empty %}
+                            <li>No author</li>
+                            {% endfor %}
+                            <li>ISBN: {{ title.isbn13 }}</li>
+                            <li><img src="{{ title.clusive.thumbnail }}" alt="Cover of {{ title.title }}"</img></li>
+                            {% if title.clusive.more %}
+                            <li id="short_{{ title.bookshareId }}">
+                                Description: {{ title.clusive.synopsis }} &hellip;
+                                <button id="more_{{title.bookshareId}}" class="link-btn more-or-less">See more</button>
+                            </li>
+                            <li id="long_{{ title.bookshareId }}" style="display: none">
+                                Description: {{ title.synopsis }}
+                                <button id="less_{{title.bookshareId}}" class="link-btn more-or-less">See less</button>
+                            </li>
+                            {% else %}
+                            <li id="long_{{ title.bookshareId }}">
+                               Description: {{ title.synopsis }}
+                            </li>
+                            {% endif %}
+                        </ul>
+                    </div>
+                    
+                    <div class="form-group">
+                        <input type="submit" class="btn btn-secondary btn-small" value="Import" disabled/>
+                        <a href="/library/bookshare-results/{{ previous }}" class="btn btn-link btn-small">Cancel</a>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+    <!-- TODO: Move this moreOrLess() script to a common location, such as frontend/js/... -->
+    <script type="text/javascript">
+        clusive.moreOrLess = function (event) {
+            const id = (event.target.id).replace(/(more_|less_)/, '');
+            const action = (event.target.id).replace(id, '');
+            if (action === "more_") {
+                var elementToShow = document.getElementById(`long_${id}`);
+                var elementToHide = document.getElementById(`short_${id}`);
+            }
+            else if (action === "less_") {
+                var elementToHide = document.getElementById(`long_${id}`);
+                var elementToShow = document.getElementById(`short_${id}`);
+            }
+            elementToHide.style.display = 'none';
+            elementToShow.style.display = 'block';
+            event.stopPropagation();
```

Comment:
Most likely want click event to be able to propagate - it is possible that something like event logging could be hooked into these controls in the future. Event logging uses delegated events, `event.stopPropagation()` would block this functionality.

Reply:
Right.  I've deleted the line.